desktop/core/src/desktop/lib/metrics/views.py

```diff
@@ -35,4 +35,4 @@ def index(request):
       'timestamp': datetime.datetime.utcnow().isoformat(),
       'metric': global_registry().dump_metrics(),
   }
-  return JsonResponse(rep, indent=indent)
```

Comment:
Sorry I forgot, how does it change the output? (would you have an example of before/after?)

This is to make sure that we don't break existing tool relying on the current format of the metrics. If the format changes, we should probably add a new GET parameter (e.g. if request.GET.get('json_dumps_params') ) and return the new response if there.